Framework/MDAlgorithms/src/MDNormDirectSC.cpp

```diff
@@ -405,29 +405,29 @@ void MDNormDirectSC::cacheDimensionXValues() {
                                (PhysicalConstants::h * PhysicalConstants::h);
   if (!m_hIntegrated) {
     auto &hDim = *m_normWS->getDimension(m_hIdx);
-    m_hX.resize(hDim.getNBins());
+    m_hX.resize(hDim.getNBins()+1);
```

Comment:
What do you think about adding another member function to `IMDDimension` that returns the `getNBins() + 1`? Perhaps `IMDDImension::getNPoints()`

Reply:
I will do it as a separate issue. Would a name like `getNBoundaries` be more accurate?

Reply:
`getNBoundaries` sounds better to me. Adding this should be one line of code. There's no reason to move it to a later PR.